contract/contract.go

```diff
@@ -14,28 +14,30 @@ import (
 	"github.com/minio/sha256-simd"
 )
 
-type loadedReply struct {
-	tx  *types.Tx
-	ex  *executor
-	err error
-}
+/* The preloadWorker optimizes the execution time by preloading the next transaction while executing the current transaction */
 
-type preLoadReq struct {
-	preLoadService int
+type preloadRequest struct {
+	preloadService int
```

Comment:
it is better to change preloadServiceType. ( BlockFactory and ChainService )
it would good also that add specific type, not int.

```
type preloadServiceType int

const (
	BlockFactory preloadServiceType = iota
	ChainService
	MaxVmService
)
```

Reply:
Done

Reply:
I made a change on [this commit](https://github.com/aergoio/aergo/pull/219/commits/c0ff64b7210298590386490a704f8d10acd4b871) but it affects many places, included passing this variable to and from C code

Then I decided to let this PR be just about renaming and comments

You can check the errors here:

```
Error: contract/contract.go:194:4: cannot use preloadService (variable of type preloadServiceType) as type int in argument to NewVmContext
Error: contract/contract.go:330:4: cannot use request.preloadService (variable of type preloadServiceType) as type int in argument to NewVmContext
Error: contract/vm.go:147:19: cannot use ChainService (constant 1 of type preloadServiceType) as type int in assignment
Error: contract/vm.go:1161:12: cannot use ChainService + 1 (constant 2 of type preloadServiceType) as type int in assignment
Error: contract/vm_callback.go:1498:15: invalid operation: service < BlockFactory (mismatched types _Ctype_int and preloadServiceType)
Error: contract/vm_callback.go:1510:16: invalid operation: service != BlockFactory (mismatched types _Ctype_int and preloadServiceType)
```

Reply:
ok.